base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -458,11 +477,31 @@ public Double getInstantFuelConsumption() {
         return SdlDataTypeConverter.objectToDouble(object);
     }
 
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1.0, please see climateData.
+     * {"num_min_value": -40.0, "num_max_value": 100.0}
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
+     */
+    @Deprecated
     public OnVehicleData setExternalTemperature(Double externalTemperature) {
         setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
         return this;
     }
 
+    /**
+     * Gets the externalTemperature.
+     *
+     * @return Boolean The external temperature in degrees celsius. This parameter is deprecated starting RPC

```

Comment:
@kostyaBoss  I tried with the generator again and it gave me `Float` for line 498.  Since the `Float` is an issue, you can use `Double`.

On line 498 you can change `Boolean` to `Double`.